src/reducers/shell.ts

```diff
@@ -47,7 +47,8 @@ export const ReservedPresetNames = {
   Error: 'Error',
   Idle: 'Idle',
   Img: 'Img',
-  Document: 'Document'
+  Document: 'Document',
+  MiniAudioUI: 'MiniAudioUI'
```

Comment:
theres no other way except adding the preset to the core ?

Reply:
It is just in the const with is globally used across our plugins